addon/devtools/loader.js

```diff
@@ -1,2 +1,38 @@
 /* global chrome */
-chrome.devtools.panels.create('Autoconsent', '/icons/cookie.png', "/devtools/panel.html")
+chrome.devtools.panels.create(
+  "Autoconsent",
+  "/icons/cookie.png",
+  "/devtools/panel.html"
+);
+
+if (chrome.devtools?.recorder) {
+  class MyPlugin {
+    stringify(recording) {
+      const autoconsentSteps = recording.steps
+        .filter((step) => ["click"].includes(step.type))
+        .map(({ selectors }) => {
+          return {
+            waitForThenClick: selectors,
+          };
+        });
+      return Promise.resolve(JSON.stringify(autoconsentSteps, undefined, 4));
+    }
+    stringifyStep(step) {
```

Comment:
Is this function actually used? As far as I can tell, `stringify()` is called all the time, and the API docs are vague

Reply:
This function is required, and used to export a single step from the recorded flow.